dluhc-component-library/src/components/policyForm/PolicyForm.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from "preact/hooks";
+import { Input, Textarea } from "../formComponents";
+import MapComponent from "../maps/MapComponent";
+import {
+  BOUNDARY_KEY,
+  DESCRIPTION_KEY,
+  FORM_lABELS,
+  INITIAL_FORM_STATE,
+  REQUIREMENTS,
+  TITLE_KEY,
+} from "./constants";
+import { FormState, FormValue } from "./types";
+
+const PolicyForm = () => {
+  const [formState, setFormState] = useState<FormState>(INITIAL_FORM_STATE);
+
+  const handleValueChange = (key: string, value: FormValue) => {
```

Comment:
I tried to get something similar in the Timetable form, how does typescript know `FormValue` is a string for the inputs and not a boundary?

Reply:
Discussed offline. It's because I'm spreading the state in and typescript isnt smart enough to work out that the key might not match the right form value meaning this doesnt give full type safety.